src/commands/RPG/util.ts

```diff
@@ -0,0 +1,81 @@
+export function cyrb53(str: string, seed = 0): number {
+  // Hash the input string to int
+  let h1 = 0xdeadbeef ^ seed,
+    h2 = 0x41c6ce57 ^ seed
+  for (let i = 0, ch; i < str.length; i++) {
+    ch = str.charCodeAt(i)
+    h1 = Math.imul(h1 ^ ch, 2654435761)
+    h2 = Math.imul(h2 ^ ch, 1597334677)
+  }
+  h1 = Math.imul(h1 ^ (h1 >>> 16), 2246822507) ^ Math.imul(h2 ^ (h2 >>> 13), 3266489909)
+  h2 = Math.imul(h2 ^ (h2 >>> 16), 2246822507) ^ Math.imul(h1 ^ (h1 >>> 13), 3266489909)
+  return 4294967296 * (2097151 & h2) + (h1 >>> 0)
+}
+
+export function mulberry32(a: number): () => number {
+  // a is a seed, returns a simple seeded RNG function from then.
+  return function () {
+    let t = (a += 0x6d2b79f5)
+    t = Math.imul(t ^ (t >>> 15), t | 1)
+    t ^= t + Math.imul(t ^ (t >>> 7), t | 61)
+    return ((t ^ (t >>> 14)) >>> 0) / 4294967296
+  }
+}
+
+export function getRandomElement(a: Array<any>) {
+  // returns a random element from a using Math.random()
+  return a[Math.floor(Math.random() * a.length)]
+}
+
+export function getSeededRandomElement(a: Array<any>, rng: () => number) {
+  // returns a random element from a,
+  // calling rng argument to get random number.
+  return a[Math.floor(rng() * a.length)]
+}
```

Comment:
To avoid reusing code you can make use of a default param like so:

```suggestion
export function getRandomElement(a: any[], seed: number = Math.random()) {
  // returns a random element with given seed; Math.random() by default
  return a[Math.floor(seed * a.length)]
}
```

Then where you called `getSeededRandomElement` you'd just call `getRandomElement(arr, someRngFunc())`

Reply:
Should also mention you can keep the param as an `rng` param and you'd just change the param to look like `rng: () => number = Math.random` to default it to use that rng func

Reply:
So this was actually a deliberate decision to have two separate methods. Initially I had a default parameter setup, but it can hide errors that makes the character generation unstable. 

I decided there is a very real difference between wanting a seeded roll, and wanting a non-seeded roll. I decided that should be reflected in the method name, rather than in an optional argument that could be forgotten. With an optional parameter it would be easy to call a non-seeded RNG when expecting a seeded one. With the two functions it is not so easy.

I'm happy to change it if you feel strongly, but otherwise I would rather take the small code duplication in order to make it more self-documenting.

Reply:
I'm okay with explicitly different named functions for differentiation. I still think that one should be the "base" function and the other should call that one. so either have `getRandomElement` have no 2nd param but call `getSeededRandomElement` with the param being `Math.random` or vice versa. If the actual logic is doing the same thing it makes future maintainability harder/more tedious when you have to make the same change 2 places when it could just be once.